chef_master/source/install_chef_automate.rst

```diff
@@ -145,6 +150,8 @@ If you don't have an existing Chef server installed and configured, the steps be
 Chef Server Installation
 ------------------------------------------------------
 
+.. note:: The standalone Chef server installation is not needed for most workflows. Only install the standalone Chef server if your workflow depends on it.
```

Comment:
I'm confused by this note...the standalone process here is basically the same as the one in the Chef server guide:

https://docs.chef.io/install_server.html#standalone

I think standalone Chef servers are very common..